test/unit/chat/dateFolder.test.ts

```diff
@@ -1,87 +1,54 @@
 import * as RdfLib from 'rdflib'
-
 import DateFolder from '../../../src/chat/dateFolder'
-jest.mock('rdflib')
+
 jest.mock('solid-auth-client')
 
 describe('DateFolder', () => {
   it('exists', () => {
-    expect(new DateFolder({ dir: () => {} })).toBeInstanceOf(DateFolder)
+    expect(new DateFolder(RdfLib.sym('https://domain.tld/dir/'))).toBeInstanceOf(DateFolder)
```

Comment:
remove `RdfLib.` like in other files

Reply:
Ok, that will require a bit more changes, as functions in dateFolder.js is dependent on the rdflib being exposed on the window-object. But it's good to write this away, so will do that.